Dockerfile

```
@@ -0,0 +1,22 @@
+FROM fedora:29 as BUILDENV
+RUN dnf --assumeyes install redhat-rpm-config
```

```suggestion
RUN dnf clean all
RUN dnf --assumeyes install redhat-rpm-config
```
In the dockerhub env, following error occured,

```
Step 1/21 : FROM fedora:29 as BUILDENV
---> d09302f77cfc
Step 2/21 : RUN dnf --assumeyes install redhat-rpm-config
---> Running in ad570b336c2e
Fedora Modular 29 - x86_64 3.0 kB/s | 3.4 kB 00:01
Failed to synchronize cache for repo 'fedora-modular'
Error: Failed to synchronize cache for repo 'fedora-modular'
```